html/js/display-map.js

```diff
@@ -18,10 +18,10 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-/*global L */
-/*exported displayMap*/
+import * as L from './leaflet-src.esm.js';
+import { MarkerClusterGroup } from './leaflet.markercluster.js';
```

Comment:
I tried to test it locally, but I'm getting an error when displaying search results on a map:

display-map.js:22 Uncaught SyntaxError: The requested module './leaflet.markercluster.js' does not provide an export named 'MarkerClusterGroup' (at display-map.js:22:10)

Reply:
Sorry, I'll take a look

Reply:
Looks like it's not super easy to use Leaflet.markercluster as an ES6 module. I'm checking whether forking the library to do so is feasible. Making the PR a draft until then.

Reply:
There's now a [good fork](https://github.com/KristjanESPERANTO/Leaflet.markercluster) that supports Leaflet v2. Updated this PR to use it.